packages/core-contracts/contracts/proxy/UUPSImplementation.sol

```diff
@@ -8,7 +8,7 @@ import "./ProxyStorage.sol";
 contract UUPSImplementation is ProxyStorage, ContractUtil, CommonErrors {
     event Upgraded(address implementation);
 
-    function upgradeTo(address newImplementation) public virtual {
```

Comment:
I see a problem here thinking ahead, upgradeTo should exist (at least as virtual or in an interface) to enable again the anti-bricking protection (since we need to simulate an upgrade from the interface)

Reply:
Yeah, until then its not needed. Following the practice of only adding things when needed.